modules/rosa-autoscaling-nodes.adoc

```diff
@@ -47,3 +51,18 @@ Enable autoscaling on a machine pool with the ID `mp1` on a cluster named `myclu
 ----
 $ rosa edit machinepool --cluster=mycluster mp1 --enable-autoscaling --min-replicas=2 --max-replicas=5
 ----
+. To disable autoscaling on a machine pool, enter the following command:
++
+[source,terminal]
+----
+$ rosa edit machinepool --cluster=<cluster_name> <machinepool_ID> --enable-autoscaling=false --replicas=<number>
```

Comment:
It's best to explain the variables that you use in a command. How do you get the number of replicas to set?